blog.qmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Teaching R and Python for Geographic Data: Shared Experience and Lessons Learned"
+# Render on save: true
+---
+
+# Introduction
+
+In this blog post, we talk about our experience teaching R and Python for geocomputation, broadly defined as [follows](https://r.geocompx.org/intro#what-is-geocomputation):
+
+> Working with geographic data in a computational way, focusing on code, reproducibility and modularity.
+
+There are are two major motivations for this post:
+
+- To share our experience teaching R and Python for geocomputation, at the OpenGeoHub Summer School in Poznan, Poland
```

Comment:
you can write "Poznań" (with ń) if you want

Reply:
Cool! https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request